daft/expressions/expressions.py

```diff
@@ -2597,6 +2597,18 @@ def slice(self, start: int | Expression, end: int | Expression) -> Expression:
         end_expr = Expression._to_expression(end)
         return Expression._from_pyexpr(self._expr.list_slice(start_expr._expr, end_expr._expr))
 
+    def chunk(self, size: int) -> Expression:
```

Comment:
We should also add this into our Expressions docs: `docs/source/api_docs/expressions.rst`

Reply:
Gotcha. I didn't do this for the slice expression too, so I'll add it at the same time.